tests/src/kakarot/instructions/test_stop_and_arithmetic_operations.cairo

```diff
@@ -23,24 +23,29 @@ func test__exec_add__should_add_0_and_1{
     // Given
     alloc_locals;
     let (bytecode) = alloc();
-    let stack: model.Stack* = Stack.init();
+    let stack = Stack.init();
 
-    let stack: model.Stack* = Stack.push(stack, Uint256(1, 0));
-    let stack: model.Stack* = Stack.push(stack, Uint256(2, 0));
-    let stack: model.Stack* = Stack.push(stack, Uint256(3, 0));
+    tempvar item_2 = new Uint256(1, 0);
+    tempvar item_1 = new Uint256(2, 0);
+    tempvar item_0 = new Uint256(3, 0);
```

Comment:
ahh so the difference here is whether or not we use a "new" keyword, yielding either the pointer or the value?